docs/acp/user/develop-an-app/develop-an-app.md

```diff
@@ -98,6 +111,12 @@ docker/sdk testing codecept run
 
 You will now see that your app boilerplate code is up and running.
 
+### Step validation
+
+- Make sure that all the command are executed without errors and the tests are successfully passed.
+- Check that docker containers are started by using the `docker/sdk ps` command.
+- Make sure that you added `my-app.de.spryker.local` into yours `/etc/hosts`
```

Comment:
I think one can check this is done correctly by pinging the domain and making sure it resolves to the localhost.